index.html

```diff
@@ -6153,6 +6200,10 @@ <h3><dfn>Execute Script</dfn></h3>
 <p>The <a>remote end steps</a> are:
 
 <ol>
+ <li><p>Let <var>name</var> be the result of
+  <a>getting a property</a> named <var>name</var>
+  from the <var>parameters</var>.
+
  <li><p>Let <var>body</var> and <var>arguments</var> be the result of
   <a>trying</a> to <a>extract the script arguments from a request</a>
   with argument <var>parameters</var>.
```

Comment:
And somewhere in there (and below, under Execute Async Script) we need to replace the _body_ with the pinned script, right?